### PR TITLE
express logger integration

### PR DESCRIPTION
**Pull request by AlKass**

### CODE REVIEW

**Comment by fadeenk** [comment]
Also why not use the [official logger](https://github.com/expressjs/morgan) when running in dev mode?? and in production we can pipe the logs into a file 😄

**Comment by AlKass** [comment]
I don't know if the official logger can be manipulated. But if it can then sure

**Comment by fadeenk** [comment]
The official logger (morgan) is fully customizble

File: server/app.js [hunk 1]
```diff
@@ -9,6 +10,7 @@ const app = express()
 app.use(cors())
 app.use(express.static(path.resolve(__dirname, '..', 'build')))
 app.use('/public', express.static(path.resolve(__dirname, '..', 'public')))
+app.use(morgan('combined'))
```
Comment (fadeenk) [inline server/app.js]: do we want to use combined?? I like dev and short but thats just a preference. it looks good though

Comment (AlKass) [inline server/app.js]: We'll use combined for debugging purposes and will define our own format later in production

Comment (fadeenk) [inline server/app.js]: It sounds perfect to me

**Comment by AlhasanIQ** [comment]
i do like both of your ideas, 
but i had a good experience with winston in the past,
could you take a look at it ?

**Pull request merged**